orchagent/nhgorch.cpp

```diff
@@ -84,12 +84,18 @@ void NhgOrch::doTask(Consumer& consumer)
                         is_recursive = true;
                 }
             }
-
+            /* A NHG should not have both regular(ip/alias) and recursive fields */
+            if (is_recursive && (!ips.empty() || !aliases.empty()))
+            {
+                SWSS_LOG_ERROR("Nexthop group %s has both regular(ip/alias) and recursive fields", index.c_str());
```

Comment:
Can we move this to info log ?

Reply:
Since the nexthop-group is not created due to this check, it's better to log this as error.